src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import javax.swing.*;
+
+public class Square extends Figure {
```

Comment:
The class definition should not start with an empty line. Please remove the empty line above the class declaration.